MembraneRTC/Sources/MembraneRTC/MembraneRTC.swift

```diff
@@ -817,9 +909,15 @@ extension MembraneRTC {
 
         pc.setRemoteDescription(description, completionHandler: { error in
             guard let err = error else {
+                [TrackEncoding.h, TrackEncoding.m, TrackEncoding.l].forEach({ encoding in
+                    self.localTracks.forEach({track in
+                        if(track.rtcTrack().kind == "video" && (track as? LocalVideoTrack)?.simulcastConfig.activeEncodings.contains(encoding) != true) {
+                            self.disableTrackEncoding(trackId: track.rtcTrack().trackId, encoding: encoding)
+                        }
+                    })
+                })
```

Comment:
Could we write a comment that this is a workaround until server supports `~` in SDP?